buildSrc/subprojects/integration-testing/src/main/kotlin/org/gradle/gradlebuild/test/integrationtests/IntegrationTest.kt

```diff
@@ -32,6 +33,9 @@ import kotlin.concurrent.timerTask
 @CacheableTask
 open class IntegrationTest : DistributionTest() {
 
+    @get:InputDirectory
```

Comment:
The `PathSensitivity` annotation is missing. Actually, running `buildSrc:validateTaskProperties` should fail. Does it?

Reply:
Yes... but why this is not included in `sanityCheck`? https://e.grdev.net/s/vmlof4uv5uyba/timeline?taskFilter=validateTaskProperties

Reply:
It is called `validatePlugins` now: https://e.grdev.net/s/vmlof4uv5uyba/timeline?details=yawmr44sl4754&taskFilter=validatePlugins

Reply:
Moreover, this in in `buildSrc`, so the check should run when building `buildSrc`. Do we not run the `buildSrc` tests any more on CI?

Reply:
We do, the `Compile All` job should be priming the cache for `buildSrc`